docs/next/.versioned_content/_versions_with_static_links.json

```diff
@@ -0,0 +1,298 @@
+    "version": "1.0.11"
+  },
+  {
+    "url": "https://dagster-git-release-1012-elementl.vercel.app",
+    "version": "1.0.12"
+  },
+  {
+    "url": "https://dagster-git-release-1013-elementl.vercel.app",
+    "version": "1.0.13"
+  },
+  {
+    "url": "https://dagster-git-release-1014-elementl.vercel.app",
+    "version": "1.0.14"
+  },
+  {
+    "url": "https://dagster-git-release-1015-elementl.vercel.app",
+    "version": "1.0.15"
+  },
+  {
+    "url": "https://dagster-git-release-1016-elementl.vercel.app",
+    "version": "1.0.16"
+  },
+  {
+    "url": "https://dagster-git-release-1017-elementl.vercel.app",
+    "version": "1.0.17"
+  },
+  {
+    "url": "https://dagster-git-release-111-elementl.vercel.app",
+    "version": "1.1.1"
+  },
+  {
+    "url": "https://dagster-git-release-112-elementl.vercel.app",
+    "version": "1.1.2"
+  },
+  {
+    "url": "https://dagster-git-release-113-elementl.vercel.app",
+    "version": "1.1.3"
+  },
+  {
+    "url": "https://dagster-git-release-114-elementl.vercel.app",
+    "version": "1.1.4"
+  },
+  {
+    "url": "https://dagster-git-release-115-elementl.vercel.app",
+    "version": "1.1.5"
+  },
+  {
+    "url": "https://dagster-git-release-116-elementl.vercel.app",
+    "version": "1.1.6"
+  },
+  {
+    "url": "https://dagster-git-release-117-elementl.vercel.app",
+    "version": "1.1.7"
+  },
+  {
+    "url": "https://dagster-git-release-118-elementl.vercel.app",
+    "version": "1.1.8"
+  },
+  {
+    "url": "https://dagster-git-release-119-elementl.vercel.app",
+    "version": "1.1.9"
+  },
+  {
+    "url": "https://dagster-git-release-1110-elementl.vercel.app",
+    "version": "1.1.10"
+  },
+  {
+    "url": "https://dagster-git-release-1111-elementl.vercel.app",
+    "version": "1.1.11"
+  },
+  {
+    "url": "https://dagster-git-release-1113-elementl.vercel.app",
+    "version": "1.1.13"
+  },
+  {
+    "url": "https://dagster-git-release-1114-elementl.vercel.app",
+    "version": "1.1.14"
+  },
+  {
+    "url": "https://dagster-git-release-1115-elementl.vercel.app",
+    "version": "1.1.15"
+  },
+  {
+    "url": "https://dagster-git-release-1117-elementl.vercel.app",
+    "version": "1.1.17"
+  },
+  {
+    "url": "https://dagster-git-release-1118-elementl.vercel.app",
+    "version": "1.1.18"
+  },
+  {
+    "url": "https://dagster-git-release-1119-elementl.vercel.app",
+    "version": "1.1.19"
+  },
+  {
+    "url": "https://dagster-git-release-1120-elementl.vercel.app",
+    "version": "1.1.20"
+  },
+  {
+    "url": "https://dagster-git-release-1121-elementl.vercel.app",
+    "version": "1.1.21"
+  },
+  {
+    "url": "https://dagster-git-release-120-elementl.vercel.app",
+    "version": "1.2.0"
+  },
+  {
+    "url": "https://dagster-git-release-121-elementl.vercel.app",
+    "version": "1.2.1"
+  },
+  {
+    "url": "https://dagster-git-release-122-elementl.vercel.app",
+    "version": "1.2.2"
+  },
+  {
+    "url": "https://dagster-git-release-123-elementl.vercel.app",
```

Comment:
This version and all those before it have a few issues/inconsistencies with 1.2.4 and newer:

1. No "This is an outdated version" callout
2. Version menu defaults to `master` instead of the version it's for (ex: `1.2.3`)

I'm not sure if it's possible to change both of these things, but I do think the lack of the "This is old" callout is an issue that could cause confusion. I'm not banking on folks paying too much attention to the URL they're looking at.

![Screenshot 2023-08-10 at 1 03 04 PM](https://github.com/dagster-io/dagster/assets/16615212/65f7a599-a99a-4dd7-b643-55e91d116b41)

Reply:
those were the previous deployment that we built during each release. So maybe we could use https://legacy-versioned-docs.dagster.dagster-docs.io/getting-started for all the older versions for consistency?

Reply:
updated. https://yuhan--docs-no-version-version-dropdown.dagster.dagster-docs.io/getting-started
let me know what you think!